distribution/ddf-common/src/main/resources/etc/org.ops4j.pax.logging.cfg

```diff
@@ -17,113 +17,207 @@
+
+# org.apache.lucene
+log4j2.logger.org_apache_lucene.name = org.apache.lucene
+log4j2.logger.org_apache_lucene.level = INFO
+log4j2.logger.org_apache_lucene.additivity = false
+log4j2.logger.org_apache_lucene.appenderRef.solr.ref = solr
+log4j2.logger.org_apache_lucene.appenderRef.syslog.ref = syslog
+log4j2.logger.org_apache_lucene.appenderRef.osgi-platformLogging.ref = osgi-platformLogging
 
 # CXF and Solr logging is verbose.  Default setting to WARN.  This can be changed in the karaf console.
-log4j.logger.org.apache.cxf = WARN
-log4j.logger.lux.solr = WARN
-log4j.logger.org.ops4j.pax.web.jsp = WARN
-log4j.logger.org.apache.aries.spifly = WARN
-log4j.logger.org.apache.cxf.phase.PhaseInterceptorChain = ERROR
+log4j2.logger.org_apache_cxf.name = org.apache.cxf
+log4j2.logger.org_apache_cxf.level = WARN
+log4j2.logger.lux_solr.name = lux.solr
+log4j2.logger.lux_solr.level = WARN
+log4j2.logger.org_ops4j_pax_web_jsp.name = org.ops4j.pax.web.jsp
+log4j2.logger.org_ops4j_pax_web_jsp.level = WARN
+log4j2.logger.org_apache_aries_spifly.name = org.apache.aries.spifly
+log4j2.logger.org_apache_aries_spifly.level = WARN
+log4j2.logger.org_apache_cxf_jaxrs_impl_WebApplicationExceptionMapper.name = org.apache.cxf.jaxrs.impl.WebApplicationExceptionMapper
+log4j2.logger.org_apache_cxf_jaxrs_impl_WebApplicationExceptionMapper.level = ERROR
+log4j2.logger.org_apache_cxf_phase_PhaseInterceptorChain.name = org.apache.cxf.phase.PhaseInterceptorChain
+log4j2.logger.org_apache_cxf_phase_PhaseInterceptorChain.level = ERROR
+
+# Appenders configuration
+
+# osgi-platformLogging
+log4j2.appender.osgi-platformLogging.type = PaxOsgi
+log4j2.appender.osgi-platformLogging.name = osgi-platformLogging
+log4j2.appender.osgi-platformLogging.filter = org.codice.ddf.platform.logging.LoggingService
+
+# osgi-all
+log4j2.appender.osgi-all.type = PaxOsgi
+log4j2.appender.osgi-all.name = osgi-all
+log4j2.appender.osgi-all.filter = *
+
+# syslog
+log4j2.appender.syslog.type = Syslog
+log4j2.appender.syslog.name = syslog
+log4j2.appender.syslog.facility = AUTH
+log4j2.appender.syslog.host = localhost
+log4j2.appender.syslog.protocol = UDP
+log4j2.appender.syslog.port = 514
+
+# stdout
+log4j2.appender.stdout.type = Console
+log4j2.appender.stdout.name = stdout
+log4j2.appender.stdout.target = SYSTEM_OUT
+log4j2.appender.stdout.layout.type = PatternLayout
+log4j2.appender.stdout.layout.pattern = %d{ABSOLUTE} | ${log4j2.pattern}
+
+# out
+log4j2.appender.out.type = RollingFile
+log4j2.appender.out.name = out
+log4j2.appender.out.fileName = ${karaf.data}/log/ddf.log
+log4j2.appender.out.filePattern = ${karaf.data}/log/ddf.log-%d{yyyy-MM-dd-HH}-%i.log.gz
+log4j2.appender.out.append = true
+log4j2.appender.out.layout.type = PatternLayout
+log4j2.appender.out.layout.pattern = %d{ISO8601} | ${log4j2.pattern}
+log4j2.appender.out.policies.type = Policies
+log4j2.appender.out.policies.size.type = SizeBasedTriggeringPolicy
+log4j2.appender.out.policies.size.size = 200MB
```

Comment:
These values were updated to match the values in https://github.com/codice/ddf/blob/ddf-2.10.2/distribution/ddf-common/src/main/resources/etc/log4j2.config.xml#L51 that are used in a hardened system. @coyotesqrl, do you know why these values were increased from `20MB` to `200 MB` in the log4j2 xml file?